packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart

```diff
@@ -0,0 +1,492 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
Add a TODO that we should merge this code with the basic debugger variable view. This code is in many ways better than the existing debugger view. Having two inconsistent ways of showing debugger object views will be a bit confusing for users.